apidocs/describe-builtin-models.js

```diff
@@ -5,6 +5,7 @@
 
 /* eslint quotes: ["error", "single"] */
 var fs = require('fs');
+var g = require('strong-globalize')();
```

Comment:
This should be inside the `try`/`catch` below where `require('loopback')` is done. This is what allows `apidocs.strongloop.com` to work without installing any dependencies.

This line is the root cause of strongloop-internal/apidocs.strongloop.com#56 :-(